tests/unit/plugins/service/test_init.py

```diff
@@ -61,19 +61,15 @@ def exp_sub(consumer_config):
     return f'projects/{consumer_config["project"]}/subscriptions/{subscription}'
 
 
-# For some reason, the event loop for this test leaks over to
-# tests.unit.plugins.service.test_event_consumer:test_gpsthread_add_task so
-# there's a warning of not awaiting the coroutine (which is expected).
-# Threads + asyncio + testing is hard.
-@pytest.mark.filterwarnings('ignore:coroutine')
 @pytest.mark.parametrize('local,provide_loop,topic,sub', [
     (True, True, 'a-topic',
      'projects/test-example/subscriptions/a-subscription'),
     (False, False, 'projects/test-example/topics/a-topic', 'a-subscription'),
 ])
-def test_get_event_consumer(local, provide_loop, topic, sub, consumer_config,
-                            exp_topic, auth_client, exp_sub, subscriber_client,
-                            emulator, monkeypatch, event_loop):
+def test_get_event_consumer(
+        local, provide_loop, topic, sub, consumer_config, exp_topic,
```

Comment:
This change from a linter? This style change goes against the rest of the style of new lines for func args.